data/decks/phonon.json

```diff
@@ -0,0 +1,216 @@
+{
+	"id": "phonon",
+	"season": 6,
+	"display_name": "Phonon",
+	"cardback": "cardback_phonon.jpg",
+	"exceed_cost": 1,
+	"character_action_default": [{
+		"gauge_cost": 0,
+		"force_cost": 0,
+		"effect": {
+			"effect_type": "strike"
+		}
+	}],
+	"character_action_exceeded": [
+		{
+			"gauge_cost": 1,
+			"force_cost": 0,
+			"effect": {
+				"condition": "last_turn_was_strike",
+				"effect_type": "choice",
+				"choice": [
+					{ "effect_type": "close", "amount": 1, "and": { "effect_type": "strike" } },
+					{ "effect_type": "retreat", "amount": 1, "and": { "effect_type": "strike" } }
+				],
+				"negative_condition_effect": {
+					"is_negative_effect": true,
+					"effect_type": "strike"
+				}
+			}
+		},
+		{
+			"gauge_cost": 0,
+			"force_cost": 0,
+			"min_hand_size": 1,
+			"action_name": "Revert",
+			"effect": {
+				"effect_type": "gauge_from_hand",
+				"min_amount": 1,
+				"max_amount": 1,
+				"and": {
+					"effect_type": "revert",
+					"and": {
+						"effect_type": "take_bonus_actions",
+						"amount": 1
+					}
+				}
+			}
+		}
+	],
+	"ability_effects": [
+		{
+			"timing": "during_strike",
+			"condition": "used_character_action",
+			"condition_details": ["default", 0],
+			"effect_type": "pass",
+			"and": {
+				"character_effect": true,
+				"condition": "last_turn_was_strike",
+				"effect_type": "powerup",
+				"amount": 1
+			}
+		},
+		{
+			"timing": "during_strike",
```

Comment:
Any particular reason for not using a negative_effect for this? mostly just curious because I do kinda feel like this way is more readable

Reply:
This way is just way more readable.